backend/census_historical_migration/sac_general_lib/general_information.py

```diff
@@ -0,0 +1,153 @@
+    FormFieldMap("auditee_address_line_1", "street1", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_city", "city", FormFieldInDissem, None, str),
+    FormFieldMap(
+        "auditee_contact_name", "auditeecontact", FormFieldInDissem, None, str
+    ),
+    FormFieldMap("auditee_contact_title", "auditeetitle", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_email", "auditeeemail", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_name", "auditeename", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_phone", "auditeephone", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_state", "state", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_uei", "uei", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_zip", "zipcode", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_address_line_1", "cpastreet1", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_city", "cpacity", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_contact_name", "cpacontact", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_contact_title", "cpatitle", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_country", "cpacountry", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_ein", "auditor_ein", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_ein_not_an_ssn_attestation", None, None, True, bool),
+    FormFieldMap(
+        "auditor_email", "cpaemail", FormFieldInDissem, "noemailfound@noemail.com", str
+    ),  # FIXME: Do we want to keep `noemailfound@noemail.com`?
+    FormFieldMap("auditor_firm_name", "cpafirmname", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_phone", "cpaphone", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_state", "cpastate", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_zip", "cpazipcode", FormFieldInDissem, None, str),
+    FormFieldMap("ein", "ein", "auditee_ein", None, str),
+    FormFieldMap("ein_not_an_ssn_attestation", None, None, True, bool),
+    FormFieldMap("is_usa_based", None, None, True, bool),
+    FormFieldMap("met_spending_threshold", None, None, True, bool),
+    FormFieldMap("multiple_eins_covered", "multipleeins", None, None, bool),
+    FormFieldMap(
+        "multiple_ueis_covered", "multipleueis", "is_additional_ueis", None, bool
+    ),
+    FormFieldMap(
+        "user_provided_organization_type", None, "entity_type", "unknown", str
+    ),  # FIXME: There is no in_db mapping ?
+    FormFieldMap("secondary_auditors_exist", "multiple_cpas", None, None, bool),
+]
+
+
+def _period_covered(s):
+    period_dict = {"A": "annual", "B": "biennial", "O": "other"}
+    if s not in period_dict:
+        raise KeyError(f"Key '{s}' not found in period coverage mapping")
+    return period_dict[s]
+
+
+def _census_audit_type(s):
+    audit_type_dict = {
+        "S": "single-audit",
+        "P": "program-specific",
+        "A": "alternative-compliance-engagement",
+    }

```

Comment:
Similar here, could we define a const outside of the method?

Reply:
Similar to my comment above.